composer.json

```diff
@@ -1,4 +1,45 @@
 {
+  "name": "civicrm/civicrm-core",
+  "description": "Open source constituent relationship management for non-profits, NGOs and advocacy organizations.",
+  "type": "library",
+  "authors": [
```

Comment:
I like that idea perhaps having the Core team + community, i think the idea is these are the people actively maintaining the thing